WebARonARKit/WebARonARKit.js

```diff
@@ -882,7 +882,8 @@
       }
     } else if (event.type == "planesremoved") {
       for (var i = 0; i < event.planes.length; i++) {
-        WebARonARKitVRDisplay.planes_.delete(event.planes[i]);
+        var plane = event.planes[i];
```

Comment:
It is a comment from a previous change but I think events should be fired after the internal structure is changed, not after (referring to the dispatchEvent call at the beginning on this function.

Reply:
done.